app/static/recipe.js

```diff
@@ -74,3 +74,61 @@ $(function(){
         runRecipe(url, form.serialize());
     });
 });
+
+
+$(function() {
+
+    $('input[id^="datepicker"]').daterangepicker({
+
+        "singleDatePicker": true,
+        "showDropdowns": true,
+        "showCustomRangeLabel": false,
+        "locale": {
+            "format": "YYYY-MM-DD",
+            "separator": " - ",
+            "applyLabel": "Apply",
+            "cancelLabel": "Cancel",
+            "fromLabel": "From",
+            "toLabel": "To",
+            "customRangeLabel": "Custom",
+            "weekLabel": "W",
+            "daysOfWeek": [
+                "Su",
+                "Mo",
+                "Tu",
+                "We",
+                "Th",
+                "Fr",
+                "Sa"
+            ],
+            "monthNames": [
+                "January",
+                "February",
+                "March",
+                "April",
+                "May",
+                "June",
+                "July",
+                "August",
+                "September",
+                "October",
+                "November",
+                "December"
```

Comment:
I'm a bit confused why you need to list these. Does the datepicker widget not have these built-in?